integration/insert_compat_test.go

```diff
@@ -23,12 +23,14 @@ import (
 	"go.mongodb.org/mongo-driver/bson"
 	"go.mongodb.org/mongo-driver/bson/primitive"
 	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
 
 	"github.com/FerretDB/FerretDB/integration/setup"
 )
 
 type insertCompatTestCase struct {
-	insert     bson.D                   // required
+	insert     []any                    // required, slice of bson.D to be insert
+	ordered    bool                     // defaults to false
```

Comment:
As we use this struct for other insert test cases that are not `ordered` related, we could also make this a `*bool` and handle nil values as `true`. That's only suggestion and it's non-critical, though currently, because of defaulting to false, we run all other test cases as unordered and it could be misleading in the future, while adding other `insertMany` test cases

Reply:
Yeah, I was thinking about it, I like that suggestion :)

Reply:
fixed